### ISSUE TITLE
Title of guidelines §2.2.8 does not precisely match content

### ISSUE THREAD

**Issue by agnasso** [TEIC/TEI#2415]
§2.2.8 is titled "Computer Files Derived from Other Computer Files" but it actually only covers the case of TEI files derived from another TEI file (or, to be annoyingly precise, "from a computer file which includes a TEI header").

Admittedly, how to proceed while creating a TEI file derived from a non-TEI file (e.g. plaintext file) is rather straightforward, but it might be opportune to explicitly address it for new users.

**Comment by sydb** [comment]
What, you mean there are non-TEI files out there? 😄
Seriously, my instinct is you have a good point, here, and re-naming the section is in order. While it might also be worth having a section devoted to metadata for non-TEI digital files, my instinct is to wait until the CMC chapter is out, first, and see what remains to be addressed.

**Comment by trishaoconnor** [comment]
Hello, @agnasso. Here is a suggestion for rewording the title and the first two sentences of the description in the TEI Guidelines for 2.2.8:

Updated title: "TEI documents derived from other TEI documents"

Updated content: (Reduced the original two-sentence opening description to one sentence.)
```
When a TEI document (call it B) is derived not from a
printed source but from another TEI document (call it
A), then the five sections
of A's file header will need to be incorporated into the new
header for B as indicated below:
```

**Comment by trishaoconnor** [comment]
Closed #2415 as completed.
